moped-data-events/activity_log/app.py

```diff
@@ -0,0 +1,72 @@
+#
+# Resolves the location for a crash.
```

Comment:
seems like some VZ specific copy/pasta here.

Reply:
Will fix that @mateoclarke